cli.py

```diff
@@ -1,4 +1,27 @@
 import click
+import os
+from ml.base import MLConfig, MLEstimator
 
-def main():
-    pass
+_CONFIG_PATH = os.path.join(os.getcwd(), 'configs/')
+
+@click.command()
+@click.argument('action')
+@click.option('--config', help='The config file')
+def main(action, config):
+    settings = MLConfig.from_file(os.path.join(_CONFIG_PATH, config))
+    model = MLEstimator.from_config(settings)
+    def predict():
+        pass
+    def train():
+        pass
+    action_case = {
```

Comment:
so 'train' and 'predict' should be commands rather than an arg named as 'action'